packages/react-instantsearch/rollup.config.js

```diff
@@ -10,7 +10,8 @@ const clear = x => x.filter(Boolean);
 
 const version = process.env.VERSION || 'UNRELEASED';
 const algolia = '© Algolia, inc.';
-const link = 'https://community.algolia.com/react-instantsearch';
+const link =
+  'https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/react/';
```

Comment:
Shouldn't that rather be the [GitHub link](https://github.com/algolia/react-instantsearch)?

That's what we did for InstantSearch.js https://github.com/algolia/instantsearch.js/blob/21bc4d5ab27d8c5bc1fab161e8fe5549addd555a/scripts/rollup.config.js#L11

Reply:
Not sure it really matters, but I can update it.